examples/fscloud/main.tf

```diff
@@ -149,6 +153,28 @@ module "cbr_account_level" {
       zone_ids = [module.cbr_zone_operator_ips.zone_id]
     }]
   })
+
+  appconfig_aggregator_service_access = {
```

Comment:
Suggest to add a comment with link to https://cloud.ibm.com/docs/app-configuration?topic=app-configuration-ac-configuration-aggregator#ac-list-of-services-configaggregator for any future maintainer